src/plugins/condition/components/Criterion.vue

```diff
@@ -77,8 +89,10 @@ export default {
     data() {
         return {
             telemetryMetadata: {},
+            telemetryMetadataOptions: {},
             operations: OPERATIONS,
-            isInputOperation: false,
+            filteredOps: [],
+            inputCount: 0,
```

Comment:
Is this just the length of an array inputs? you should be able to just access the inputs length directly. Or, if you want to be safe you can wrap this in a computed prop like so,

`inputCount() {
  return inputs && inputs.length;
}`

Reply:
It is the value of the inputCount property of the selected comparison object.

Reply:
thanks for clarifying @JoelMcKinnon